components/Navigation/pen-tool.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const PenToolIcon = () => {
+  // eslint-disable-next-line max-len
+  return (
```

Comment:
Same as above, no need for return statement
```suggestion
  // eslint-disable-next-line max-len
const PenToolIcon = () => (
```